.github/workflows/collector-slim.yml

```
@@ -20,44 +20,69 @@ env:
   RHACS_ENG_IMAGE: quay.io/rhacs-eng/collector:${{ inputs.collector-tag }}
 
 jobs:
+  set-environment:
+    name: Set environment variables
+    runs-on: ubuntu-latest
+    outputs:
+      build-builder-image: ${{ steps.set-env.outputs.build-builder-image }}
+      collector-builder-tag: ${{ steps.set-env.outputs.collector-builder-tag }}
+
+    steps:
+      - name: Checks PR, main and release branches
+        id: set-env
+        run: |
+
+          if [[ "${{ github.event_name != 'pull_request' }}" ]]; then
+            echo 'build-builder-image=true' >> "$GITHUB_OUTPUT"
+            echo "collector-builder-tag=${DEFAULT_BUILDER_TAG}" >> "$GITHUB_OUTPUT"
+
+          elif [[ "${{ github.event_name == 'pull_request' }}" ]]; then
```

Similar to the previous comment
```suggestion
          elif [[ "${{ github.event_name }}" == 'pull_request' ]]; then
```